subprojects/core/src/main/java/org/gradle/process/internal/worker/WorkerLoggingSerializer.java

```diff
@@ -39,10 +40,11 @@ public static SerializerRegistry create() {
         BaseSerializerFactory factory = new BaseSerializerFactory();
         Serializer<LogLevel> logLevelSerializer = factory.getSerializerFor(LogLevel.class);
         Serializer<Throwable> throwableSerializer = factory.getSerializerFor(Throwable.class);
+        Serializer<OperationIdentifier> operationIdSerializer = factory.getSerializerFor(OperationIdentifier.class);
```

Comment:
This class could use some simple test coverage.  At least a test that it returns a registry that can serialize the expected types.